topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md

```diff
@@ -387,20 +392,20 @@ the dataset number 28.
 ## Getting an overview of your samples' chromatograms
 
 You may be interested in getting an overview of what your samples' chromatograms look like, for example to see if some of
-your samples have distinct overall characteristics, for example unexpected chromatographic peaks, or huge overall intensity.
+your samples have distinct overall characteristics, *e.g.* unexpected chromatographic peaks or huge overall intensity.
 
 You can use the *sampleMedata* file we previously uploaded to add some group colours to your samples when visualising your chromatograms.
 The tool automatically takes the second column as colour groups when a file is provided.
 
 Note that you can also check the chromatograms at any moment during the workflow, in particular at the following steps:
- - After **MSnbase readMSData** {% icon tool %}: to help you to define retention time ranges that you may want to discard from the very beginning (*"Specta Filters"* in **findChromPeaks** {% icon tool %})
- - After **adjustRtime** {% icon tool %}: to check the result of the correction (rerun *adjustRtime* with other settings)
+ - After **MSnbase readMSData** {% icon tool %} to help you to define retention time ranges that you may want to discard from the very beginning (*"Specta Filters"* in **findChromPeaks** {% icon tool %})
```

Comment:
I cannot find: findChromPeaks in usegalaxy.eu; maybe this one: xcms findChromPeaks (xcmsSet); the parameter is probably called 'spectra filters'

Reply:
Indeed! Thanks.
See  8a02dcc